app/src/main/java/de/danoeh/antennapod/fragment/EpisodesListFragment.java

```diff
@@ -180,7 +185,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             ((SimpleItemAnimator) animator).setSupportsChangeAnimations(false);
         }
 
-        SwipeRefreshLayout swipeRefreshLayout = root.findViewById(R.id.swipeRefresh);
+        swipeRefreshLayout = root.findViewById(R.id.swipeRefresh);
         swipeRefreshLayout.setDistanceToTriggerSync(getResources().getInteger(R.integer.swipe_refresh_distance));
         swipeRefreshLayout.setOnRefreshListener(() -> {
```

Comment:
The code 2 lines below this hides the refresh layout again after a few milliseconds. This is no longer needed. As it currently is, the PR hides the refresh layout after a few milliseconds, so there is no indication  whether a refresh is in progress.